src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
+    _sdClass, 'ui',
+    useKeyOnly(left, 'left aligned'),
+    useKeyOnly(center, 'center aligned'),
+    useKeyOnly(right, 'right aligned'),
+    useKeyOnly(justified, 'justified'),
+    useKeyOnly(dividing, 'dividing'),
+    useKeyOnly(block, 'block'),
+    useKeyOnly(attached, 'attached'),
+    useKeyOnly(floating, 'floating'),
```

Comment:
'floating' (as per Label.js) or 'floated' as you suggest below?

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 17:44
>
> In src/elements/Header/_Header.js 
> <https://github.com/TechnologyAdvice/stardust/pull/278#discussion_r68425954>:
>
> > + const {
> > + _sdClass, _headerElement,
> > + left, center, right, justified, dividing, block, attached, floating,
> > + icon, image, children, className,
> > + } = props
> > +
> > + const classes = cx(
> > + _sdClass, 'ui',
> > + useKeyOnly(left, 'left aligned'),
> > + useKeyOnly(center, 'center aligned'),
> > + useKeyOnly(right, 'right aligned'),
> > + useKeyOnly(justified, 'justified'),
> > + useKeyOnly(dividing, 'dividing'),
> > + useKeyOnly(block, 'block'),
> > + useKeyOnly(attached, 'attached'),
> > + useKeyOnly(floating, 'floating'),
>
> The floating <http://semantic-ui.com/elements/header#floating> option 
> appears as |right floated| and |left floated| html classes. These 
> would be expressed in the API as:
>
> <Header floated='right' />
> <Header floated='left' />
>
> Then in className buildup as:
>
> cx(
> useValueAndKey(floated, 'floated'),
> )
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/278/files/4317290b6fd2e4980cfd74e736287438f69d4c6c#r68425954>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHszhX1VolKzPQjewIlQ3B9TaGhfWnks5qPAlmgaJpZM4I9saB>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>